### DISCUSSION TITLE
Linux distribution with hyprdots

### DISCUSSION THREAD

**Discussion by DiegoSof2** [prasanthrangan/hyprdots#875] [Ideas]
Would it be possible to create an iso with pure archlinux + hyprdots containing all the packages,

I find installing hyprdots a bit time consuming and it is also possible that the packages required to install hyrdots may be broken in the future,

Would it then be possible to create a stardalone distribution or make an entire backup of the system that can be installed on other machines?

**Comment by kRHYME7** [comment]
@DiegoSof2  That would be very cool! 

For now, the best way is this https://github.com/prasanthrangan/hyprdots/discussions/634 or use chaotic AUR before installation of hyprdots. This will ensure that most packages are pre-compiled and have a bigger chance not to be broken.

**Comment by DiegoSof2** [comment]
> @DiegoSof2 That would be very cool!
> 
> For now, the best way is this #634 or use chaotic AUR before installation of hyprdots. This will ensure that most packages are pre-compiled and have a bigger chance not to be broken.

I will check
